org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/scheduler/SchedulerStateEditor.java

```diff
@@ -349,6 +350,25 @@ else if (schedulerState == IServerManager.SERVER_STATE_STOPPED)
 		
 		return label;
 	}
+	
+	@Override
+	public String getDisplayTextForGridView(GridRowCtx gridRowCtx, Object value) {
+		schedulerState=0;
+		MScheduler scheduler = new MScheduler(Env.getCtx(), Integer.parseInt(value.toString()), null);
```

Comment:
I think it is better to be more defensive here and handle the possibility that value can be null and parseInt can throw exception.